### PR TITLE
Add (usage of) aliases for ContentValidationListener

### PR DESCRIPTION
**Pull request by rkeet**
Provide a narrative description of what you are trying to accomplish:

- [x] Are you creating a new feature?
  - [x] Why is the new feature needed? What purpose does it serve?

Usage of an alias allows users to provide own implementation of ContentValidationListener

  - [x] How will users use the new feature?

Users might wish to add/remove/alter ContentValidationListener implementation (like me :) )

  - [x] Base your feature on the `develop` branch, and submit against that branch.

Done

  - [x] Add only one feature per pull request; split multiple features over multiple pull requests

Done -> Adding aliases. Implementing single usage of aliases in Module.php.

  - [-] Add tests for the new feature.

Not a feature, no change in functionality.

  - [x] Add documentation for the new feature.

Updated with changes

  - [-] Add a `CHANGELOG.md` entry for the new feature.

Not a feature, just expanding and making optional which one to use, without modification of current implementation. **Would this require changelog entry?**

### CODE REVIEW

**Comment by rkeet** [comment]
Created this PR so that I (and others when they know about it) can choose to implement their own/a modified ContentValidationListener. 

I've also run into [this issue (#59)](https://github.com/zfcampus/zf-content-validation/issues/59). I'll be creating a feature branch with a fix for it shortly. However, if that [new PR] does not pass the wishes of the maintainers for this repo, then maybe this PR can at least give the option for others to implement what they need outside of this repo.

**Comment by rkeet** [comment]
another wee ping @weierophinney

File: src/Module.php [hunk 1]
```diff
@@ -21,6 +21,6 @@ public function onBootstrap(MvcEvent $e)
         $events   = $app->getEventManager();
         $services = $app->getServiceManager();
 
-        $services->get(ContentValidationListener::class)->attach($events);
+        $services->get('ContentValidationListener')->attach($events);
```
Comment (weierophinney) [inline src/Module.php]: The aliases existed originally due to when ZF2 was originally developed and released. We tended to use just the class name at the time, minus the namespace; zend-servicemanager was also case insensitive. As we prepared for ZF3, we realized that these names were easy to mistype, and started moving to using `::class` notation, as static analyzers and IDEs will flag incorrectly typed class names when you use it, making them more safe.

The aliases we provide at this point are a legacy: they exist only to allow retrieval using the v2 naming conventions. The service itself **MUST** be named after the class, as that's the future direction.

What this means is that _in your own code_ you can override the entry such that the factory assigned will return an alternate implementation. If you do that, any of the aliases or the canonical name will return your variant.

As such, I am going to close this issue, as there is already a mechanism for supplying an alternative listener.

Comment (rkeet) [reply src/Module.php]: Fair enough, thanks for having a look though :)

**Pull request closed without merging**